src/wallet/rpcwallet.cpp

```diff
@@ -3501,6 +3192,10 @@ UniValue spendmany(const UniValue& params, bool fHelp) {
 
     LOCK2(cs_main, pwalletMain->cs_wallet);
 
+    // Only account "" have sigma coins.
+    if (!params[0].isNull() && !params[0].get_str().empty())
```

Comment:
I think we should remove null checking since it confusing due to this one it not optional parameter.

Reply:
Shouldn't this also be in a `try` block? If `get_str()` fails it will throw

Reply:
Agree, the latter check is enough. We are already assuming that each param element is the correct type and exists